packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -111,7 +148,7 @@ public void onMethodCall(MethodCall call, Result result) {
    * completed (either successfully or in error). This class provides no synchronization consructs
    * to guarantee such behavior; callers are responsible for providing such guarantees.
    */
-  public static final class Delegate {
+  public static final class DelegateImpl implements Delegate {
```

Comment:
This is a breaking change.  Rather than having to bump a minor version number, you could keep this `Delegate` and make the interface (I can't believe I'm saying this...) `IDelegate`.  Then again, I hate myself for having said that, so maybe you just bump the minor version number 😄-- wdyt?

Reply:
As discussed offline, it's probably not worth a breaking change to change the name of the Delegate class so I renamed the interface to IDelegate.